kythe/typescript/kythe.ts

```diff
@@ -0,0 +1,211 @@
+/**
+ * @fileoverview TypeScript typings of the Kythe schema.
```

Comment:
'typings' has a specific meaning in TS, it's about defining types for a JS library, which I think doesn't apply here.

Reply:
Fixed.